include/bluetooth/mesh/gen_loc_cli.h

```diff
@@ -100,6 +100,9 @@ struct bt_mesh_loc_cli {
  * function will return, and the response will be passed to the
  * @ref bt_mesh_loc_cli_handlers::global_status callback.
  *
+ * @note Blocking calls must be made from the main cooperative thread only. Do not use in
```

Comment:
> 
Blocking calls must be made from the threads, blocking of which doesn't disrupt functionality of Zephyr system services. Do not use blocking calls from interrupt handlers (for example button handlers) and kernel services (for example the system work queue handlers).